src/InstrumentationEngine.Lib/LoggerService.cpp

```diff
@@ -133,8 +133,13 @@ void CLoggerService::SetLogToDebugPort(_In_ bool enable)
     RecalculateLoggingFlags();
 }
 
-HRESULT CLoggerService::Initialize()
+HRESULT CLoggerService::Initialize(_In_ std::function<void(const LoggingFlags&)> callback)
```

Comment:
>In_ std::function<void(const LoggingFlags&)> callback [](start = 36, length = 53)

 I think Justin was looking for something that would be possible for instrumentation methods to register to as well. That would imply adding something to the idl, and making it late registration. I'm fin with this for now, though as well. I wouldn't want to block this check in.

Reply:
True, I didn't want to modify the API surface yet, we can definitely address these callbacks holistically in the future.